js/.eslintrc

```diff
@@ -169,7 +169,7 @@
         "prefer-const": 0,
         "prefer-reflect": 0,
         "prefer-spread": 2,
-        "prefer-template": 2,
+        "prefer-template": 0, // not replace content in string
```

Comment:
I changed this line because template string not working with QUnit test, so all my test cases failed